content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -522,24 +522,33 @@ This means that support is still available but will be removed in a future Kuber
 * [`rbd`](/docs/concepts/storage/volumes/#rbd) - Rados Block Device (RBD) volume
   (**deprecated** in v1.28)
 
-Older versions of Kubernetes also supported the following in-tree PersistentVolume types:
+The following PersistentVolume types are supported through the `CSIMigration` feature gate and require the corresponding CSI driver to be installed in the cluster. Kubernetes offers built-in APIs to facilitate their use.
 
 * [`awsElasticBlockStore`](/docs/concepts/storage/volumes/#awselasticblockstore) - AWS Elastic Block Store (EBS)
-  (**not available** in v1.27)
+  &mdash; but for Kubernetes version 1.27 and onwards, this volume type is **only** available via a separate CSI driver 
+  as Kubernetes' core no longer provides built-in support. The Kubernetes project provides an equivalent
+  CSI driver, [AWS EBS](https://github.com/kubernetes-sigs/aws-ebs-csi-driver).
 * [`azureDisk`](/docs/concepts/storage/volumes/#azuredisk) - Azure Disk
-  (**not available** in v1.27)
+  &mdash; but for Kubernetes version 1.27 and onwards, this volume type is **only** available via a separate CSI driver
+  as Kubernetes' core no longer provides built-in support. The Kubernetes project provides an equivalent
+  CSI driver,
+  [Azure Disk](https://github.com/kubernetes-sigs/azuredisk-csi-driver).
 * [`cinder`](/docs/concepts/storage/volumes/#cinder) - Cinder (OpenStack block storage)
-  (**not available** in v1.26)
-* `photonPersistentDisk` - Photon controller persistent disk.
-  (**not available** starting v1.15)
-* `scaleIO` - ScaleIO volume.
-  (**not available** starting v1.21)
-* `flocker` - Flocker storage.
-  (**not available** starting v1.25)
-* `quobyte` - Quobyte volume.
-  (**not available** starting v1.25)
-* `storageos` - StorageOS volume.
-  (**not available** starting v1.25)
+    &mdash; but for Kubernetes version 1.26 and onwards, this volume type is **only** available via a separate CSI driver 
```

Comment:
CSI migration is GA since 1.24

Reply:
The documentation is capturing the version since the use of the CSI driver was mandatory. So 1.24 is correct, right?

Reply:
Updated the version in the latest commit.